readme.md

```diff
@@ -490,14 +490,14 @@ A more complete TypeScript guide is [here](docs/guides/typescript.md).
 
 ## Best practices
 
-- You may wonder how to organize your code for better maintenance: [Splitting the store into separate slices](docs/typescript.md#slices-pattern).
-- Recommended usage for this unopinionated library: [Flux inspired practice](docs/flux-inspired-practice.md).
-- [Calling actions outside a React event handler in pre React 18](docs/event-handler-in-pre-react-18.md).
+- You may wonder how to organize your code for better maintenance: [Splitting the store into separate slices](./docs/guides/typescript.md#slices-pattern).
+- Recommended usage for this unopinionated library: [Flux inspired practice](./docs/guides/flux-inspired-practice.md).
+- [Calling actions outside a React event handler in pre React 18](./docs/guides/event-handler-in-pre-react-18.md).
 - Testing: [Wiki page](https://github.com/pmndrs/zustand/wiki/Testing).
```

Comment:
Should this link directly to the guide? eg: `./docs/guides/testing.mdx`

Reply:
You're right! Fixed!